### PR TITLE
Added Assignments and Identity Service Example

### PR DESCRIPTION
**Pull request by Radhika-Camunda**
Added Assignments and Identity Service Example #issue-618995437

### CODE REVIEW

File: content/reference/bpmn20/tasks/user-task.md [hunk 1]
```diff
@@ -262,7 +262,13 @@ is intentional, in order to avoid creating event loops.
 Although the Camunda engine provides an identity management component, which is exposed through the IdentityService, it does not check whether a provided user is known by the identity component. This allows integration of the engine with existing identity management solutions when it is embedded into an application.
 
 However, note that you can use the identity service in a service / bean or listener to query your user repository if this is useful to you.
-
+Provided user's identity can check with identity services
+See the following example:
+```java
+IdentityService identityService = Context.getProcessEngineConfiguration().getIdentityService();
```
Comment (tasso94) [inline content/reference/bpmn20/tasks/user-task.md:268]: Can we remove this line? With the `Context` class, it uses internal API, which we do not recommend.

File: content/reference/bpmn20/tasks/user-task.md [hunk 2]
```diff
@@ -262,7 +262,13 @@ is intentional, in order to avoid creating event loops.
...
+List users =identityService.createUserQuery().memberOfGroup("management").list();
```
Comment (tasso94) [inline content/reference/bpmn20/tasks/user-task.md:269]:
```suggestion
List<User> users = identityService.createUserQuery().memberOfGroup("management").list();
```

File: content/reference/bpmn20/tasks/user-task.md (hunk 1 above)
Comment (Radhika-Camunda) [reply content/reference/bpmn20/tasks/user-task.md:268]: Sure  I will remove below line.
IdentityService identityService = Context.getProcessEngineConfiguration().getIdentityService();
or can we re-frame this with below code snippet?
ProcessEngine processEngine = delegateTask.getProcessEngine();
IdentityService identityService = processEngine.getIdentityService();

Comment (tasso94) [reply content/reference/bpmn20/tasks/user-task.md]: Sure, you can rephrase it like that. Let me know if you wanna do it or not.

Comment (Radhika-Camunda) [reply content/reference/bpmn20/tasks/user-task.md]: Thanks Tasso .. I have incorporated the changes. Please review and approve the pull request

**Comment by tasso94** [comment]
Hi @Radhika-Camunda,

Thanks again for your contribution!

I changed the wording a bit and formatted the example according to our code style guidelines.

Best,
Tassilo

**Pull request merged**